website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "version-17.0.0",
```

Comment:
Should probably match the `v17.0.0` format

Reply:
Definitely.   🤦🏻

Reply:
bd5d456956